websites/J/Jisho/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "SagaFreya",
+    "id": "316314134990028810"
+  },
+  "service": "Jisho",
+  "description": {
+    "en": "Jisho is a powerful Japanese-English dictionary. It lets you find words, kanji, example sentences and more quickly and easily."
+  },
+  "url": [
+    "jisho.org",
+	"classic.jisho.org"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/nAtEXHe.png",
+  "thumbnail": "https://i.imgur.com/ecPtY82.png",
```

Comment:
Please change the thumbnail to a screenshot of the website or a promotional image